etc/schemas/arduino-library-properties-definitions-schema.json

```diff
@@ -91,7 +91,7 @@
                 "$ref": "#/definitions/propertiesObjects/name/specification/object"
               },
               {
-                "maxLength": 16
+                "maxLength": 32
```

Comment:
```suggestion
                "maxLength": 36
```
I apologize for being slow to review. The Tooling Team discussed this internally and decided that it should be 36.

Reply:
> Can you share the rationale for "36"?

The suggestion comes from @91volt supported by a visive test on the IDE:

![image](https://user-images.githubusercontent.com/423515/225626180-d802bfd7-78e8-40cf-a44a-4d36699ef96d.png)

I'm fine with `32` too, no strong opinion on this.